src/update.ts

```diff
@@ -61,9 +75,9 @@ export class UpdateQuery<
       | null,
     onRejected?: ((reason: any) => Result2 | PromiseLike<Result2>) | undefined | null,
   ): Promise<Result1 | Result2> {
-    const queryState = createQueryState(this.tokens);
+    const queryState = createQueryState(this.toTokens());
 
-    return this.queryExecutor(queryState.text.join(` `), queryState.parameters)
+    return this.queryExecutor(`${queryState.text.join(` `)}`, queryState.parameters)
```

Comment:
This line does not need to change, the string interpolation is no longer needed.